ArWikiCats/new/resolve_films_bots/resolve_films_labels.py

```
@@ -156,6 +191,21 @@ def _make_bot() -> MultiDataFormatterBase:
     return bot
 
 
+def fix_keys(category: str) -> str:
+    """Fix known issues in category keys."""
+    normalized_text = category.lower().replace("category:", " ").strip()
```

Ruff (0.14.10)

Local variable `normalized_text` is assigned to but never used

Remove assignment to unused variable `normalized_text`

(F841)